src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+import javafx.application.Application;
+
```

Comment:
It would be helpful if you could add Javadocs to explain the purpose of the Launcher class!